### PR TITLE
Update readme with final and dev versions

### PR DESCRIPTION
**Pull request by karesti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 [![Infinispan](https://design.jboss.org/infinispan/logo/final/PNG/infinispan_logo_rgb_lightbluewhite_darkblue_600px.png)](https://infinispan.org/)
 
-[![Version](https://img.shields.io/maven-central/v/org.infinispan/infinispan-core?logo=apache-maven&style=for-the-badge)](https://search.maven.org/artifact/org.infinispan/infinispan-core)
+[![Version Final](https://img.shields.io/maven-central/v/org.infinispan/infinispan-core?logo=apache-maven&style=for-the-badge&versionPrefix=15.0)](https://search.maven.org/artifact/org.infinispan/infinispan-core)
+[![Version Dev](https://img.shields.io/maven-central/v/org.infinispan/infinispan-core?logo=apache-maven&style=for-the-badge&versionPrefix=15.1)](https://search.maven.org/artifact/org.infinispan/infinispan-core)
```
Comment (pruivo) [inline README.md:4]: Why do we have the Final and Dev links pointing to the same location? 
Shouldn't be https://search.maven.org/artifact/org.infinispan/infinispan-core/15.0.10.Final/jar and https://search.maven.org/artifact/org.infinispan/infinispan-core/15.1.0.Dev04.Final/jar

Comment (karesti) [reply README.md:4]: The prefix brings the latest one of each

Comment (pruivo) [reply README.md:4]: I know, but both points to `https://search.maven.org/artifact/org.infinispan/infinispan-core`

**Comment by ryanemerson** [comment]
Thanks @karesti

**Comment by alanfx** [comment]
Hey @karesti the `README.md` has two links that look correct (One for `v15.0.10.Final` and one for `v15.1.0.Dev04`), but both links end up at the `v15.1.0.Dev04` version on Maven Central.

**Comment by karesti** [comment]
@alanfx @pruivo the link points to the latest release whatever that one is, we can't be changing the readme on every minor to point to it. I opened a Pr to land into the versions tab instead

**Comment by alanfx** [comment]
Hey @karesti, I guess it would help to have the exact current versions in some variable that could be put into the README.md, but I guess that doesn't exist. I think it is kind of better the way it is now, because at least the page that shows the DEV release has a popup list of versions to pick from?

**Pull request merged**